source/agora/node/Registry.d

```diff
@@ -115,8 +115,8 @@ public class NameRegistry: NameRegistryAPI
     private void collectStats (Collector collector)
     {
         RegistryStats stats;
-        stats.registry_validator_record_count = this.zones[this.validators].count();
-        stats.registry_flash_record_count = this.zones[this.flash].count();
+        stats.registry_validator_record_count = (this.validators in this.zones).count();
```

Comment:
I think changes like these are not needed. There won't be any copying unless you do;

`ValueType value = AA[Key];`

Feel free to revert such usages or merge.